docs/src/man/basics.md

```diff
@@ -0,0 +1,1388 @@
+# First Steps with DataFrames.jl
+
+The first and important part to work with DataFrames.jl, is to install the `DataFrames`
+package which is available thorugh the Julia package system and can be installed using the
+following commands:
+
+```julia
+julia> using Pkg
+julia> Pkg.add("DataFrames")
+```
+
+Or,
+
+```julia
+julia> ] # `]` should be pressed 
+
+(@v1.6) pkg>  add DataFrames
+```
+
+If you want to make sure everything works as expected you could run the tests bundled
+with DataFrames.jl, but be warned that it will take more than 30 minutes.
+```julia
+julia> using DataFrames
+julia> using Pkg
+julia> Pkg.test("DataFrames")
+```
+
+Additionally, it is recommended to check the version installed with `st` command(this tutorial is for 1.x releases).
+```julia
+julia> ]
+
+(@v1.6) pkg> st 
+```
+
+Throughout the rest of the tutorial we will assume that you have installed DataFrames.jl package and
+have already typed `using DataFrames`.
+Let's get started by loading the DataFrames.jl package:
+```jldoctest
+julia> using DataFrames
+```
+
+The most fundamental object provided by DataFrames.jl is a `DataFrame`, where
+typically that each row is interpreted as an observation and each column as a feature.
+
+# Constructors and basic utility functions
+
+## Constructors
+
+In this section you will see many ways to create a `DataFrame` using the `DataFrame(...)` constructor.
+First, let's create an empty `DataFrame`:
+```jldoctest dataframe
+julia> using DataFrames
+
+julia> df = DataFrame()
+0×0 DataFrame
+```
+
+Or, we could call the constructor using keyword arguments to add columns to the `DataFrame`:
+```jldoctest
+julia> using DataFrames
+
+julia> df = DataFrame(A=1:3, B=5:7, fixed=1)
+3×3 DataFrame
+ Row │ A      B      fixed
+     │ Int64  Int64  Int64
+─────┼─────────────────────
+   1 │     1      5      1
+   2 │     2      6      1
+   3 │     3      7      1
+```
+note in column `:fixed` that scalars get automatically broadcasted.
+
+To move forward with the tutorial you need to install the CSV.jl package in your environment. 
+In order to do so run the following commands:
+```julia
+julia> using Pkg
+julia> Pkg.add("CSV")
+```
+Before moving further make sure you have CSV.jl in a version that is at least 0.8.4.
+
+Now, we will explore how to load a CSV file into a `DataFrame`. Unlike Python's Pandas `read_csv`
+the functions in Julia are separated into two packages - CSV.jl and DataFrames.jl. As the first
+step, you have to declare the libraries you will use. In our case CSV.jl and DataFrames.jl. In order to
+read the file in we will use the `CSV.read` function.
+
+As the first step, we have to load the libraries we will use. In our case CSV.jl and DataFrames.jl.
```

Comment:
repetition with what you have written 2 sentences above.

Reply:
But I am not getting a mistake, sorry.

Reply:
Two sentences above you have written:

> As the first step, you have to declare the libraries you will use. In our case CSV.jl and DataFrames.jl.